clusterloader2/testing/experiments/ignore_known_gce_container_restarts.yaml

```diff
@@ -9,3 +9,9 @@ RESTART_COUNT_THRESHOLD_OVERRIDES: |
   metadata-proxy: 2
   prometheus-to-sd-exporter: 2
   coredns: 2
+
```

Comment:
Do we want to specify it in ignore_known_kubemark_container_restarts.yaml as well?

Reply:
Actually - why do we have a separate list for kubemark?

Reply:
Because there are some containers that restart just in kubemark tests (for example at the time of implementing crashlooping pod detection. scheduler/controller-manager restarts only happened in kubemark). We wanted to temporarily allow it in kubemark tests and still be able to notice regressions in non-kubemark runs.